package.json

```diff
@@ -119,6 +119,7 @@
     "@babel/cli": "^7.18.6",
     "@babel/core": "^7.18.6",
     "@babel/preset-env": "^7.18.6",
+    "@cucumber/cucumber": "^9.1.2",
```

Comment:
hmmm that is an accident. shall remove

Reply:
![image](https://github.com/pact-foundation/pact-js/assets/19932401/749e6423-d5bf-4912-9721-1dbc043f331a)